### PR TITLE
Add Secrets and Values of Work Order Service

### PR DESCRIPTION
**Pull request by kaviarasu-ni**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/install-systemlink-enterprise/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

The next release of SystemLink Enterprise will add support for work orders optionally. This PR adds the required minimum values that have to be set for a successful deployment/upgrade of SLE.

### Why should this Pull Request be merged?

Adds required documentation.

### What testing has been done?

Internal testing on k8s clusters.

### CODE REVIEW

File: getting-started/templates/systemlink-values.yaml [hunk 1]
```diff
@@ -897,3 +897,51 @@ comments:
...
+##
+workorder:
+  ## Set to true to enable the deployment of the work order and lab management ui service. By default, 
+  ## this is disabled and the work order and lab management ui service are not deployed.
+  ##
+  enabled: false
+
+  ## Maximum number of work orders per create work orders request.
+  ##
+  maximumWorkOrdersPerCreateRequest: 1000
```
Comment (JD-Robertson) [inline getting-started/templates/systemlink-values.yaml:911]: Do we expect a typical customer to need to set any of these limits to something other than the default? If not, please do not include them in this file. We should avoid adding unnecessary complexity to this file.

File: getting-started/templates/systemlink-values.yaml [hunk 2]
```diff
@@ -897,3 +897,51 @@ comments:
...
   ##
   maximumCommentsPerDeleteRequest: 1000
+
+## Configuration for work order service.
+##
+workorder:
+  ## Set to true to enable the deployment of the work order and lab management ui service. By default, 
+  ## this is disabled and the work order and lab management ui service are not deployed.
+  ##
+  enabled: false
```
Comment (JD-Robertson) [inline getting-started/templates/systemlink-values.yaml:907]: Consider adding an ATTENTION tag to this value if this is a decision that should be made by the customer at installation time.

Comment (kaviarasu-ni) [reply getting-started/templates/systemlink-values.yaml:908]: Added the ATTENTION tag.

File: getting-started/templates/systemlink-values.yaml (hunk 1 above)
Comment (kaviarasu-ni) [reply getting-started/templates/systemlink-values.yaml]: The customer would not need to update these limits other than the default. Removed it.

**Review by prestwick — APPROVED** [review]

**Review by JD-Robertson — APPROVED** [review]

**Pull request merged**